### PR TITLE
New Bodhi update for 5.7.1 release

### PR DESCRIPTION
**Pull request by AdamSaleh**
Between our staging deployment and the end of freeze there were some more improvements we want to include.

### CODE REVIEW

**Comment by mattiaverga** [comment]
I think now a better strategy to get things right would be:
- prepare 5.7 release in devel branch (bump version, make release notes, etc.)
- merge devel into 5.7 with a force push

That way we can have the correct cleanup of news fragments and correct status of release notes in devel.
Future bugfix releases should be handled by cherry picking commits into release branch, in the past this was performed automatically by using mergify tags (i.e. `5.7-backports`). Then finally bump release and prepare release notes in devel and cherry pick them into release branch.

**Comment by AdamSaleh** [comment]
Ok, in that case I will point this to devel branch, because this is what it is, and after we merge is I will just force push it to 5.7 branch.

File: docs/user/release_notes.rst [hunk 1]
```diff
@@ -4,6 +4,69 @@ Release notes
 
 .. towncrier release notes start
 
+v5.7.1
+======
+This is a {major|feature|bugfix} release that adds [short summary].
```
Comment (mattiaverga) [inline docs/user/release_notes.rst]: Line needs to be fixed

File: docs/user/release_notes.rst [hunk 2]
```diff
@@ -4,6 +4,69 @@ Release notes
...
+* Staging Bodhi now uses staging Bugzilla URL for bug links (:issue:`4238`).
+* Fixed an issue where editing Updates always caused to set the request to
+  Testing (:issue:`4263`).
+
+Development improvements
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Add End of life (eol) field to the releases (:issue:`4240`).
+
+
```
Comment (mattiaverga) [inline docs/user/release_notes.rst:106]: Missing contributors

**Review by mattiaverga — COMMENTED** [review]
Also commit is missing DCO

**Comment by AdamSaleh** [comment]
Ok, fixed both dco, and the newsfiles auto-generates line and missing contributors.

**Comment by AdamSaleh** [comment]
@mattiaverga you think it looks good now? the test runner seems stuck, but the changes were just in the dco and the news-file

**Review by mattiaverga — DISMISSED** [review]
Looks good :+1:

**Comment by mattiaverga** [comment]
Oh, wait...
I missed that #4241 contains a database migration. We should add the appropriate notes to the release notes to notify of that:

> If your changes contain database migrations, you must add a file named summary.migration in the news directory, where summary is a short textual description of the change (or the issue number you’re fixing).

See https://github.com/fedora-infra/bodhi/blob/519c491fcb40f8dfe4f5c865bfce4ee63ac23cae/news/summary.migration

**Review by mattiaverga — APPROVED** [review]

**Pull request merged**